mlflow/server/js/OWNERS

```diff
@@ -0,0 +1 @@
+per-file .enzymemigrationignore=*
```

Comment:
should we ignore OWNERS file in copybara settings?

Reply:
sounds good to me, how do i do that? should i also remove this file from the PR?